src/commands/mod.rs

```diff
@@ -592,6 +595,30 @@ struct UnsquashArgs {
     interactive: bool,
 }
 
+/// Sets or removes the executable bit for paths in the repo
+///
+/// Unlike the POSIX `chmod`, `jj chmod` also works on Windows, on conflicted
+/// files, and on arbitrary revisions.
+#[derive(clap::Subcommand, Clone, Debug)]
+enum ChmodSubcommand {
+    /// Make a path non-executable
+    #[command(visible_alias("n"))]
+    Normal(ChmodArgs),
+    /// Make a path executable
+    #[command(visible_alias("x"))]
+    Executable(ChmodArgs),
+}
```

Comment:
Perhaps, this is more like a positional argument of enum type than subcommand.

Reply:
Thanks. This looks better as `jj chmod --help` lists everything we need.

> it's mildly irritating that value options cannot have visible aliases that get displayed in error messages.

I don't have better idea (other than crafting our `FromStr` from scratch.)

Nit: maybe better to specify both `name` and `alias` without changing the enum variant.
```
    #[value(name = "x", alias("executable"))]
    Executable,
```

> Aside: +x is a valid value we could support, but -x isn't 

Yeah. `git` works around it by requiring `--chmod=` flag. Anyway, I don't think we have to copy the behavior of `chmod`. Most people who care about executable bit would simply do `chmod` the working-copy files.
https://www.git-scm.com/docs/git-update-index

Reply:
> Nit: maybe better to specify both name and alias without changing the enum variant.

That's exactly what I was looking for. Now I'm no longer irritated. :)

Thanks!